cms/templates/cms/home_page.html

```diff
@@ -87,9 +87,15 @@ <h3>What is the MicroMasters<span class="trademark">&reg;</span> Program Credent
               </div>
             </div>
               <p>A MicroMasters program credential is a professional and academic credential for online learners from
-                anywhere in the world. Learners who pass an integrated set of MITx graduate-level courses on edX.org,
-                and one or more proctored exams, will earn a MicroMasters program credential from MITx, and can then
-                apply for an accelerated, on campus, master’s degree program at MIT or other top universities.</p>
+                anywhere in the world. Learners who pass an integrated set of MIT graduate-level courses on edX.org,
+                and one or more proctored exams. This credential is valuable in and of itself.  Additionally,
+                credential-holders can then apply for an accelerated, on campus, master’s degree program at MIT
+                or other universities.
+                {% if benefits_url %}
+                  All MicroMasters program credential holders are also considered
+                  <a href="{{benefits_url}}">affiliates of the MIT Alumni Association</a>.
+                {% endif %}
+              </p>
```

Comment:
@pdpinch here I have added a condition to add last line only if `benefits_url` exists.